src/extension/networkInspector/views/inspectorConsoleView.ts

```diff
@@ -213,7 +214,22 @@ export class InspectorConsoleView extends InspectorView {
                 "Response Body": this.requestBodyDecoder.formatBody(response),
             },
         };
-        return networkRequestData;
+
+        if (url.search) {
+            networkRequestConsoleView.networkRequestData[
+                "Request Query Parameters"
+            ] = this.retrieveURLSearchParams(url.searchParams);
+        }
+
+        return networkRequestConsoleView;
+    }
+
+    private retrieveURLSearchParams(searchParams: URLSearchParams): Record<string, string> {
+        let formattedSearchParams = {};
```

Comment:
Could you please specify the type?

```suggestion
        let formattedSearchParams: Record<string, string> = {};
```